### ISSUE TITLE
Jaws does not read the element referenced with aria-errormessage

### ISSUE THREAD

**Issue by Wildebrew** [FreedomScientific/VFO-standards-support#83]
### Summary

Jaws does not read the contents of an element referenced with aria-errormessage when user moves focus to an invalid field (field with aria-invalid="true" and aria-errormessage="id of the error element").

To verify, 

1. head on over to [my aria-errormessage Code Pen](https://codepen.io/Wildebrew/pen/RyeGZw).
2. Enter a number that is 12 or less in the input box and click the "test it" button.
3. Focus the input field with Jaws running.

### Expected result

Jaws reads the label of the field, its value, the words "invalid" or "has error" and the error message (the target of aria-errormessage).

### Actual result

Jaws does not read the contents of the aria-errormessage target (it says "has error" in Firefox but it does not automatically read the error message in any browser).

### Example

[My aria-errormessage code pen](https://codepen.io/Wildebrew/pen/RyeGZw)

#### JAWS version and build number

Tested in various combinations of Jaws 2018 and browsers (excluding the latest builds of Firefox and Edge).

 
#### Operating System and version

Mostly N/A.

#### Browser and version:

IE11
Chrome 65
Firefox 52-ESR

**Comment by JAWS-test** [comment]
With JAWS 2019, the output is improved, but not perfect:
* Firefox and Chrome: output only on tab and not on linear read or quick navigation. In addition, output only occurs for elements that cause Form Mode, i.e. not for checkboxes.
* IE: Very irregular output
  * select: no output for tab, linear reading: "has error" (without message)
  * checkbox: Tab and linear reading: "has error" (without message)
  * input field: Tab and linear reading: "has an error" and message is displayed

**Comment by lamador** [comment]
When using the latest version of JAWS 2020 (2020.2004.66), Chrome (81.0.4044.138, Official Build, 64-bit), Firefox (76.0.1, 64-bit), and IE11 (11.778.18362.0) in Windows 10, the following was observed when testing the original Codepen example: 
- The error message was properly announced in Firefox and Chrome only when made visible.
- In IE11, the error message was announced regardless of its hidden state via CSS `display:none;`

Upon further testing using [Codepen of various aria-errormessage implementations](https://cdpn.io/TPG/debug/78108bb8423422bfcb6bca250d125de0), Chrome will announce the error message when hidden using CSS `visibility:hidden;` . IE11 always announces the presence of an error whenever `aria-errormessage` is used even when `aria-invalid="false"`, `aria-invalid` is not used, or when there is no error message text within the target container.
